src/components/cc-input-text/cc-input-text.js

```diff
@@ -39,31 +50,31 @@ const TAG_SEPARATOR = ' ';
  * @cssprop {FontSize} --cc-input-label-font-size - The font-size for the input's label (defaults: `inherit`).
  * @cssprop {FontWeight} --cc-input-label-font-weight - The font-weight for the input's label (defaults: `normal`).
  *
- * @slot error - The error message to be displayed below the `<input>` element or below the help text. Please use a `<p>` tag.
  * @slot help - The help message to be displayed right below the `<input>` element. Please use a `<p>` tag.
  */
-export class CcInputText extends LitElement {
+export class CcInputText extends CcInputElement {
 
   static get properties () {
     return {
+      ...super.properties,
       clipboard: { type: Boolean, reflect: true },
       disabled: { type: Boolean, reflect: true },
       label: { type: String },
       hiddenLabel: { type: Boolean, attribute: 'hidden-label' },
       inline: { type: Boolean, reflect: true },
       multi: { type: Boolean, reflect: true },
-      name: { type: String, reflect: true },
       placeholder: { type: String },
       readonly: { type: Boolean, reflect: true },
       required: { type: Boolean },
+      resetValue: { type: String, attribute: 'reset-value' },
```

Comment:
**❔ question:** Is there a reason for not making this property a part of `CcInputElement`?

Reply:
like for the value property, we cannot assume what type it should be.